app/models/energy/ScheduleFile.py

```diff
@@ -0,0 +1,59 @@
+"""Schedule File Schema"""
+from pydantic import BaseModel, Schema, validator, ValidationError, UrlStr, ConstrainedStr
+from typing import List, Union
+from enum import Enum
+from uuid import UUID, uuid4
+from datetime import datetime
+from app.models.energy.ScheduleBase import ScheduleContinuous, ScheduleDiscrete, ScheduleNumericType, ScheduleUnitType
+
+
+class ScheduleFile(BaseModel):
```

Comment:
I think we are also going to need an input for `is_leap_year` here in order to translate it correctly to E+. We should also add a check to ensure that someone specifies the correct number of values depending on whether it is a leap year or not.